Formula/openssl@1.1.rb

```diff
@@ -126,17 +129,38 @@ def macos_post_install
       /-----BEGIN CERTIFICATE-----.*?-----END CERTIFICATE-----/m,
     )
 
+    # Check that the certificate has not expired
     valid_certs = certs.select do |cert|
-      IO.popen("#{bin}/openssl x509 -inform pem -checkend 0 -noout >/dev/null", "w") do |openssl_io|
+      cmd = "#{bin}/openssl x509 -inform pem -checkend 0 -noout 2>&1 >/dev/null"
+      IO.popen(cmd, "w") do |openssl_io|
```

Comment:
```suggestion
      IO.popen("#{bin}/openssl x509 -inform pem -checkend 0 -noout 2>&1 >/dev/null", "w") do |openssl_io|
```

Reply:
I was going to do it this way, but it then fails the line length test

Reply:
```suggestion
      IO.popen("#{bin}/openssl x509 -inform pem -checkend 0 -noout &>/dev/null", "w") do |openssl_io|
```

may work, if not, leave it as is (but use `&>`)

Reply:
Done. 👍  Learnt some new bash shorthand